### ISSUE TITLE
NewPipe lite version

### ISSUE THREAD

**Issue by theScrabi** [TeamNewPipe/NewPipe#1224]
Some of you might already have heard that I play with the thought of releasing a lite version of NewPipe. This version would have reduced functionality so it can be put on Google Play. The fdroid version will not be touched or altered by this, and it will remain with full functionality. 

The positive thing about a lite version would be
1. We can fite copy cats on google play
2. We will gain more fame through that. Which is god because the more do use NewPipe the more will support us.

However wed have to discuss several things before we can actually do this:
- Check if we can align with the google play guidelines
- Discus if we want the app to be released under my name/google accout or if we should add a new account for TeamNewPipe
- Check if this can be aligned with our [core features](https://newpipe.schabi.org/#privacy)

**Comment by justanidea** [comment]
And dont forget to often remember them that a bigger version exists on fdroid

More other more people could support Fdroid and its great for them too :)

**Comment by Lionirdeadman** [comment]
Sure, it's free publicity *but* it may lead people to think that NewPipe doesn't have certain features

I would also think it can create more issues and require more work for development.

**Comment by speednroll** [comment]
I don't think it is necessary to think about putting a welcome warning that recommends you to use newpipe from f-droid. I think it's enough to name newpipe LITE and not find another newpipe (complete) in gplay, except in the description of the first one, heading to the Newpipe page.

**Comment by theScrabi** [comment]
We may not be allowed to add ads for fdroid on apps in the play store directly. However we can put links to our website ... there things can be explained.

Also the difference betwean beta/stable already is like a lite/pro version. It's barely 10 lines of code. Some ifs that check weaterh its release ore not and than hide/show UI ellements accordingly.

**Comment by speednroll** [comment]
Of course, I was specifically referring to advertising the Newpipe website 👍

**Comment by speednroll** [comment]
by the way, I think it would be confusing to have so many versions of Newpipe, because there is a stable version, beta, now the lite version will come out and the million question is... will newpipe lite have a beta version? 😆

**Comment by theScrabi** [comment]
Yea ... actually beta didn't play out the way I would have liked it to be. It made things more complicated than they should be. ... maybe we should go back to a moddel where we only have a release version. ... Just do the QA better this time.

**Comment by IamJithendra** [comment]
On  Thinking about Lite Version Don't Forget New pipe Original Version

**Comment by JuniorJPDJ** [comment]
@theScrabi just release beta versions as beta/unstable versions in fdroid in main app's page

**Comment by TobiGr** [comment]
> just release beta versions as beta/unstable versions in fdroid in main app's page

That's indeed a good idea.

**Comment by theScrabi** [comment]
yes it is :D

**Comment by jesus2099** [comment]
What features would be removed to make it LITE?
And then what features would remain, that would still be enhancements compared to native YouTube app?

**Comment by Julmik647** [comment]
It would be useful now, beacuse Google may add verification for apps from outside Google Play.

**Comment by Julmik647** [comment]
PeerTube and media.ccc.de can probably be unchanged, I'm not sure about Soundcloud and Bandcamp, but you may need to do these changes to YouTube:
- use offical YouTube embed for video player (youtube-nocookie.com/embed/[video id])
- make YouTube download button open separate app  or website (like en1.savefrom.net/16xF/#url=https:/youtu.be/[video id])
- background playback can probably be unchanged as some Play store apps have it
